src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -50,6 +52,21 @@ def format_historical_data(data: SnubaTSResult) -> list[TimeSeriesPoint]:
     return formatted_data
 
 
+# return the start and end indices
+def get_start_and_end(data: SnubaTSResult) -> tuple[int, int]:
```

Comment:
Maybe we make this an `_get_start_end` underscored helper?
Since this is generically named, i wouldn't want anyone else to pull and try to use this helper